presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1915,13 +1932,17 @@ private PhysicalOperation createLookupJoin(JoinNode node,
                 Optional<Symbol> buildHashSymbol,
                 LocalExecutionPlanContext context)
         {
-            // Plan probe
-            PhysicalOperation probeSource = probeNode.accept(this, context);
+            Optional<DynamicFilter> dynamicFilter = createDynamicFilter(node, probeNode, probeSymbols, buildNode, buildSymbols, context);
```

Comment:
please plug into
```
            // TODO: Execution must be plugged in here
            if (!node.getDynamicFilters().isEmpty()) {
                log.debug("[Join] Dynamic filters: %s", node.getDynamicFilters());
            }
```
within `visitJoin` code

Reply:
I moved it to `createLookupSourceFactory`, following https://github.com/prestosql/presto/pull/931#discussion_r299817744.